docs/core-concepts/core-sdk/cli/overview.md

```diff
@@ -0,0 +1,121 @@
+# .NET Core Command Line Tools (CLI)
+
+## What is the .NET CLI?
+.NET Core Command Line Interface is a new foundational cross-platform toolchain for developing 
+.NET Core applications. It is "foundational" because it is the primary layer on which other, 
+higher-level tools, such as Integrated Development Environments (IDEs), editors and 
+build orchestrators can build on. 
+
+It is also cross-platform by default and has the same surface area on each of the supported platforms. This means that
+when you learn how to use the tooling and you can use it the same way from any of the supported platforms. 
+
+## Installation
+As with any tooling, the first thing is to get the tools to your machine. Depending on your scenario, you can either 
+use the native installers to install the CLI or use the installation shell script.
+
+The native installers are primarily meant for developer's machines. The CLI is distributed using each supported platform's 
+native install mechanism, for instance DEB packages on Ubuntu or MSI bundles on Windows. These installers will install 
+and set up the environment as needed for the user to use the CLI immediately after the install. However, they also 
+require administrative privileges on the machine. You can view the installation instructions on the
+[getting started page](https://aka.ms/dotnetcoregs).
```

Comment:
nit: when linking, I'd be more specific to help build keywords for SEO purposes. So, here I'd say [.NET Core getting started page]() for example.